server/sandbox.go

```diff
@@ -111,7 +121,17 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 	g.AddBindMount(resolvPath, "/etc/resolv.conf", "ro")
 
 	labels := req.GetConfig().GetLabels()
+	labelsJSON, err := json.Marshal(labels)
+	if err != nil {
+		return nil, err
+	}
+	g.AddAnnotation("oci.labels", string(labelsJSON))
```

Comment:
Oh crap, I though I had written "ociD" with the "d" - yeah will take care of this one in a follow up